vowpalwabbit/core/src/vw.cc

```diff
@@ -276,8 +284,8 @@ std::unique_ptr<VW::workspace> VW::initialize_experimental(std::unique_ptr<confi
 }
 
 std::unique_ptr<VW::workspace> VW::initialize(std::unique_ptr<config::options_i> options,
-    std::unique_ptr<VW::io::reader> model_override_reader, bool skip_model_load,
```

Comment:
Is there a reason to remove this param and have it always skip model load?

Reply:
The only use case for this parameter is an implementation detail of seed_vw_model and I cannot think of a single end user scenario. 

Removing this parameter means we don't ever skip_model_load since there is no reason to